opensearch_py_ml/ml_commons/ml_common_utils.py

```diff
@@ -9,3 +9,4 @@
 MODEL_UPLOAD_CHUNK_SIZE = 10_000_000
 MODEL_MAX_SIZE = 4_000_000_000
 BUF_SIZE = 65536  # lets read stuff in 64kb chunks!
+TIMEOUT = 120  # timeout for synchronous method calls in seconds
```

Comment:
thanks for putting this here. Can you please make sure all the `120` number is replaced by this variable? Thanks.